packages/components/src/components/tabs/_tabs.scss

```diff
@@ -134,7 +134,8 @@
     @include carbon--breakpoint(md) {
       display: flex;
       transition: inherit;
-      overflow: visible;
+      overflow: scroll;
```

Comment:
Probably scrolling only horizontally? Another question is; Does this work with non-overlay scrollbar (Mac's Show scroll bars - Always setting in System Preferences or Windows)?

Reply:
Good catch. Updated it to `overflow-x`. Scrollbar setting doesn't seem to be affected by this change.